server/utils/s3.js

```diff
@@ -78,12 +86,6 @@ export const publicS3Endpoint = (isServerUpload?: boolean) => {
 };
 
 export const uploadToS3FromUrl = async (url: string, key: string) => {
```

Comment:
This method looks like it is only used for Avatar uploads at the moment. The potential issue here is that if I save it as a private acl, the avatars will not load. Couple of options:
1) Save avatars into the attachments table and return proxy urls like the other uploads
2) Save these avatars as special cases as public objects (for these specific objects only)

I'm leaning to option 2. Option 1 needs to rework a fair bit of stuff from adding an `Attachments.create` call, pulling team ID and user ID properties all the way through this and even potentially breaking a couple of the `NOT NULL` constraints in the attachments table. The downside to option 2 is that it won't allow users to fully lock down their *entire* s3 bucket from public access globally. But as long as `AWS_S3_ACL=private` the objects will be secure.

I think option 1 can eventually be done, but doing option 2 does not block the path for taking some more time in thinking how to implement a "proper" solution for everything which goes into the s3 bucket.

Reply:
current implementation is uploading avatars as `public-read`